CMakeLists.txt

```diff
@@ -253,6 +253,36 @@ if("${DDC_BUILD_KERNELS_SPLINES}")
   find_package(LAPACKE REQUIRED)
   target_link_libraries(DDC INTERFACE ${LAPACKE_LIBRARIES})
   target_include_directories(DDC INTERFACE ${LAPACKE_INCLUDE_DIRS})
+
```

Comment:
Is it possible to switch ginkgo and KK backend with CMake options?

Reply:
No, atm we need both to compile the splines. Maybe for another MR

Reply:
Sure